rainier-plot/src/main/scala/com/stripe/rainier/plot/EvilTraceplots.scala

```diff
@@ -0,0 +1,134 @@
+/*
+EvilTraceplots.scala
+
+Some diagnostic plots using EvilPlot
+
+ */
+
+package com.stripe.rainier.plot
+
+object EvilTraceplots {
+
+  import com.cibo.evilplot.plot._
+  import com.cibo.evilplot.colors._
+  import com.cibo.evilplot.plot.aesthetics.DefaultTheme._
+  import com.cibo.evilplot.numeric.Point
+
+  /**
+    * Autocorrelation function
+    *
+    * @param x Input data
+    * @param lm Maximum lag required
+    */
+  def acf(x: Seq[Double], lm: Int): List[Double] = {
```

Comment:
Is there a way this could share code with https://github.com/stripe/rainier/blob/master/rainier-core/src/main/scala/com/stripe/rainier/sampler/Sampler.scala#L65 ?

Reply:
In principle there ought to be, but it seems like it would involve significantly more code than it would save, so I'm inclined to leave it for now.